code/__HELPERS/icons.dm

```diff
@@ -960,4 +960,32 @@ The _flatIcons list is a cache for generated icon files.
 			mask.BecomeAlphaMask() //Make all the black pixels vanish (fully transparent), leaving only the white background pixels.
 
 		main.AddAlphaMask(mask) //Make the pixels in the main icon that are in the transparent zone of the mask icon also vanish (fully transparent).
-		return main
\ No newline at end of file
+		return main
+
+//For creating consistent icons for human looking simple animals
+/proc/get_flat_human_icon(icon_id, datum/job/J, datum/preferences/prefs, dummy_key, showDirs = cardinal, outfit_override = null)
+	var/static/list/humanoid_icon_cache = list()
+	if(!icon_id || !humanoid_icon_cache[icon_id])
+		var/mob/living/carbon/human/dummy/body = generate_or_wait_for_human_dummy(dummy_key)
+
+		if(prefs)
+			prefs.copy_to(body)
+		if(J)
+			J.equip(body, TRUE, FALSE)
+		else if(outfit_override)
+			body.equipOutfit(outfit_override, visualsOnly = TRUE)
+
+
+
+		var/icon/out_icon = icon('icons/effects/effects.dmi', "nothing")
+
+		for(var/D in showDirs)
+			body.setDir(D)

```

Comment:
no `COMPILE_OVERLAYS(body)`?

Reply:
won't compile for some reason https://puu.sh/EcMeZ/0f81724b9c.png

Reply:
You need to update to a version of 512 or something. I recommend 512.1472

Reply:
rename `__HELPERS/lists.dm` to `__HELPERS/_lists.dm

Reply:
@Aurorablade this is still relevant and will allow you to use the COMPILE_OVERLAYS thing

Reply:
yeah get on it ain a bit my email did not notify me i had a new comment..